<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphWidget</class>
 <widget class="QWidget" name="graphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>133</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Données et grandeurs à représenter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Masse (kg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_m">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Intensité de la pesanteur g (N/kg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_g">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>9.8</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Tracer :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Y"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>En fonction de :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_X"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Avec le style :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_style">
           <item>
            <property name="text">
             <string>Points seuls</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/data/icones/dots.svg</normaloff>:/data/icones/dots.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Points et lignes</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/data/icones/dots+lines.svg</normaloff>:/data/icones/dots+lines.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lignes seules</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/data/icones/lines.svg</normaloff>:/data/icones/lines.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_save_plot">
           <property name="toolTip">
            <string>Enregistrer comme image</string>
           </property>
           <property name="text">
            <string>Enregistrer</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/data/icones/folder-pictures.svg</normaloff>:/data/icones/folder-pictures.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ImageWidget" name="widget_graph" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>image_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
